Makefile

```diff
@@ -100,4 +100,13 @@ crossplane.help:
 
 help-special: crossplane.help
 
+# NOTE(hasheddan): the build submodule currently overrides XDG_CACHE_HOME in
+# order to force the Helm 3 to use the .work/helm directory. This causes Go on
+# Linux machines to use that directory as the build cache as well. We should
+# adjust this behavior in the build submodule because it is also causing Linux
+# users to duplicate their build cache, but for now we just make it easier to
+# identify its location in CI so that we cache between builds.
+go.cachedir:
```

Comment:
I've incorporated this Make target in #88, do you mind rebasing?

Reply:
Done.